jlm/util/Program.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2025 Nico Reißmann <nico.reissmann@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <jlm/util/common.hpp>
+#include <jlm/util/Program.hpp>
+#include <jlm/util/strfmt.hpp>
+
+#include <sys/wait.h>
+#include <unistd.h>
+
+namespace jlm::util
+{
+
+std::filesystem::path
```

Comment:
If you want to use path, use "bash -c" wrapper instead of searching yourself

Reply:
I was a bit uncertain here what exactly you mean, but I hope I understood you right.